Cargo.toml

```diff
@@ -57,6 +57,10 @@ telio-traversal.workspace = true
 telio-utils.workspace = true
 telio-wg.workspace = true
 
+[target.'cfg(target_os = "android")'.dependencies]
+jni = "0.19"
+rustls-platform-verifier = { git = "https://github.com/tomaszklak/rustls-platform-verifier.git", rev = "1eeed2dc3a4a7f437220875feb31e50cdec0bf07" }
```

Comment:
fork of rustls-platform-verifier is used, since original doesn't yet officially support tvos